crates/bevy_ecs/src/schedule/mod.rs

```diff
@@ -173,6 +189,30 @@ impl Schedule {
         })
     }
 
+    /// Fetches the [`Stage`] of type `T` marked with `label`, then executes `func` using the
+    /// stage as parameter.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use bevy_ecs::schedule::Schedule;
+    /// # use bevy_ecs::schedule::SystemSet;
+    /// # use bevy_ecs::schedule::SystemStage;
+    /// # use bevy_ecs::system::IntoSystem;
+    /// #
+    /// # let mut schedule = Schedule::default();
+    /// # schedule.add_stage("update", SystemStage::parallel());
```

Comment:
I'm a little nervous about using "update" here. The default stage is `CoreStage::Update`, which is confusingly similar.

Maybe "game_logic" or something?

Reply:
Keep in mind this is bevy_ecs, which is unaware of bevy_app, where the `CoreStage::Update` label and the relative stage registration happen.

From a full Bevy environment this may be concernful though. Maybe using a more generic `"my_stage"` as label could be a better fit, especially, where the user can just replace the "my_stage" with their own real stage.